packages/ERTP/src/brand.js

```diff
@@ -1,22 +1,32 @@
 // @ts-check
+import { provide } from '@agoric/store';
+import { defineDurableKind, makeKindHandle } from '@agoric/vat-data';
 import { E } from '@endo/eventual-send';
-import { Far } from '@endo/marshal';
 
 /**
+ * @param {MapStore<string,any>} issuerBaggage
  * @param {string} allegedName
  * @param {(allegedIssuer: Issuer) => boolean} isMyIssuerNow
  * @param {DisplayInfo} displayInfo
  * @returns {Brand}
  */
-export const makeBrand = (allegedName, isMyIssuerNow, displayInfo) => {
-  /** @type {Brand} */
-  const brand = Far(`${allegedName} brand`, {
-    isMyIssuer: allegedIssuerP => E.when(allegedIssuerP, isMyIssuerNow),
+export const makeDurableBrand = (
```

Comment:
is this name interim for refactoring or meant for master? generally, do you advise including `Durable` in the name or a durable kind maker even when there is no non-durable kind to disambiguate?

reading further I see the `makeBrand` is used within this method, but that conflict isn't the concern of the API. The `makeDurableBrand` name seems more appropriate to me on line 22 as the result of `defineDurableKind`.

Reply:
Interim. I changed names as I changed APIs, to keep track of where I was in refactoring. Now that I'm on the other side, all the unnecessarily complex internal names can be simplified again.

Will look again at makeBrand vs makeDurableBrand. Thanks.

Reply:
Now that this has otherwise, settled, we still need to revisit these renamings, so leaving this conversation open until then.